### ISSUE TITLE
"You've unlocked a game" screen

### ISSUE THREAD

**Issue by Jacquelyn-Rudis** [YMCACOLUMBUS/activescience-issues#41]
Shows after game is played and hypothesis made. Should this come before game is played?

**Comment by Jacquelyn-Rudis** [comment]
Also could we direct children to the game button on the bottom of the home screen at this point? Same with first vocab word in each Mission- "See your vocab words here"?

**Comment by TAGraves** [comment]
Yes, I think the right solution is to say "You've unlocked a game - play it on the My Games page" with a picture of the game controller icon.

@scottdebney what do you think of adding this to the vocab pages as well? Could require a small redesign for those pages.

**Comment by TAGraves** [comment]
@scottdebney Small redesign to two pages here.

**Comment by scottdebney** [comment]
Here's the vocab button reminder.  
[vocab.psd.zip](https://github.com/YMCACOLUMBUS/activescience-issues/files/260235/vocab.psd.zip)

**Comment by scottdebney** [comment]
Here's the play it later design.  
[play-it-later.psd.zip](https://github.com/YMCACOLUMBUS/activescience-issues/files/279769/play-it-later.psd.zip)

**Comment by Captainlonate** [comment]
Concerning the "See all your vocab words 'button'" that appears when a student is taught a new word, during a lesson:

What happens when this 'button' is clicked?
If they are in the middle of a lesson when they click this 'button', certainly we wouldn't want to interrupt the student during the lesson by ending the lesson and taking them back to the level screen.

Thanks,

**Comment by scottdebney** [comment]
Could the words just pop above the lesson?  They can click out of it with the X to get back to the Lesson?  If that's too problematic, just have it be a graphic that's not clickable.
